build.gradle

```diff
@@ -13,6 +13,8 @@ apply plugin: 'groovy'
 apply plugin: 'maven-publish'
 apply plugin: 'com.jfrog.bintray'
 apply plugin: 'org.ajoberstar.github-pages'
+apply plugin: 'idea'
```

Comment:
See comment below.

Reply:
Another time, please consider making an init script and then apply'ing that instead, then you dont have to add it to this project.

Reply:
@jutoft The declaration of the plugin doesn't really hurt until you actually run one of the IDEA tasks. Maybe there's a small impact on configuration time but it's likely insignificant.

Reply:
This is actually a pretty neat suggestion, @jutoft, thanks! I should have thought about it myself. It will allow me to add even more IntelliJ configuration to the build without having to force it on Benjamin. I will submit a PR at some time removing the idea plugin and it's configuration from the build.